edge/stack/images.go

```diff
@@ -0,0 +1,62 @@
+package stack
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/containers/image/v5/copy"
+	"github.com/containers/image/v5/signature"
+	"github.com/containers/image/v5/transports/alltransports"
+	"github.com/containers/image/v5/types"
+)
+
+// TODO: this code is copied in the portainer/portainer-ee PR too, should consolidate somewhere
+func copyImage(ctx context.Context, cacheImage, stackImage string) error {
```

Comment:
Can we rename the parameters, and maybe the function name to something more explicit?
```suggestion
func copyImage(ctx context.Context, sourceImageCacheURL, destinationImageName string) error {
```